Packs/CortexResponseAndRemediation/Playbooks/playbook-CVE-2025-31324__-_SAP_NetWeaver_Visual_Composer.yml

```diff
@@ -0,0 +1,2554 @@
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "94":
+    id: "94"
+    taskid: 3c9e5d5c-cc1f-4398-9fe7-b351d4207e4e
+    type: regular
+    task:
+      id: 3c9e5d5c-cc1f-4398-9fe7-b351d4207e4e
+      version: -1
+      name: Set Potential Files Paths
+      description: "Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.\n\nThis automation runs using the default Limited User role, unless you explicitly change the permissions.\nFor more information, see the section about permissions here:\n- For Cortex XSOAR 6 see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.x/Cortex-XSOAR-Playbook-Design-Guide/Automations \n- For Cortex XSOAR 8 Cloud see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Cloud-Documentation/Create-a-script\n- For Cortex XSOAR 8.7 On-prem see https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8.7/Cortex-XSOAR-On-prem-Documentation/Create-a-script"
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "97"
+    scriptarguments:
+      append:
+        simple: "true"
+      key:
+        simple: Potential_suspicious_files
+      value:
+        simple: ${XQLPossibleMaliciousFiles.results.action_file_path}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3770.75,
+          "y": 2530
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "95":
+    id: "95"
+    taskid: d3728ed9-ffcd-46e4-91f6-1821e4c02de9
+    type: regular
+    task:
+      id: d3728ed9-ffcd-46e4-91f6-1821e4c02de9
+      version: -1
+      name: Manual Action - Quarantine
+      description: "As a part of the playbook, we have determined potential webshells that might be in the same directory *\"...servlet_jsp\\irj\\....\"*.\nSince we can't determine with strong confidence that those files are webshells, we suggest that you investigate them and quarantine as needed. \n\nList of files:\n${Potential_suspicious_files}\n\nList of agent IDs:\n${alert.agentid}\n${alert.agentsid}"
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "31"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 3833,
+          "y": 3361
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "96":
+    id: "96"
+    taskid: 9742b3be-3d5f-40e4-854a-cd65fbd4c75e
+    type: condition
+    task:
+      id: 9742b3be-3d5f-40e4-854a-cd65fbd4c75e
+      version: -1
+      name: Is there a File path?
```

Comment:
Are we sure alert.filepath should be available for on the detector side?

Reply:
The alert wasn't triggered, so that's an educated guess.
The alert looks for suspicious creation of files in that directory - and that field is the only one that is mappet to action_file_path